docs/data-cookbooks/browse-generation.md

```diff
@@ -78,173 +76,205 @@ For this example, you are going to be adding two workflows to your Cumulus deplo
-
  If you're not ingesting mock data matching the example, or would like to use modify the example to ingest your own data please see the [build-lambda](#build-lambda) section below.    You will need to configure a different lambda entry for your lambda and utilize it in place of the ```Resource``` defined in the example workflow.
 
-#### Cumulus Configuration
+ **Please note**: ```FakeProcessing``` is the core provided browse/cmr generation lambda we're using for the example in this entry.
 
-In an editor, open app/config.yml and modify your stepFunctions key to contain the file you just created:
+#### Lambdas
 
-```yaml
-stepFunctions: !!files [
-  {some list of workflows},
-  'browseExample.yml'
-]
-```
+All lambdas utilized in this example are provided in a standard deployment of Cumulus and require no additional configuration.
 
-This will cause kes to export the workflows in the new file along with the other workflows configured for your deployment.
+If you're not ingesting mock data matching the example, or would like to use this entry to ingest your own data please see the [build-lambda](#build-lambda) section below.    You will need to add a terraform ```aws_lambda_function``` resource to your deployment.
 
-#### Lambdas
+#### Redeploy
 
-Ensure the following lambdas are in your deployment's lambdas.yml (reference the [example lambdas.yml](https://github.com/nasa/cumulus/blob/master/example/lambdas.yml)):
-
-```yaml
-DiscoverGranules:
-  handler: index.handler
-  timeout: 300
-  memory: 512
-  source: node_modules/@cumulus/discover-granules/dist/
-  useMessageAdapter: true
-  launchInVpc: true
-QueueGranules:
-  handler: index.handler
-  timeout: 300
-  source: node_modules/@cumulus/queue-granules/dist/
-  useMessageAdapter: true
-  launchInVpc: true
-SyncGranule:
-  handler: index.handler
-  timeout: 300
-  logToElasticSearch: true
-  source: node_modules/@cumulus/sync-granule/dist/
-  useMessageAdapter: true
-  launchInVpc: true
-FilesToGranules:
-  handler: index.handler
-  source: node_modules/@cumulus/files-to-granules/dist/
-  launchInVpc: true
-FakeProcessing:
-  handler: index.handler
-  source: node_modules/@cumulus/test-processing/dist/
-  useMessageAdapter: true
-  launchInVpc: true
-MoveGranules:
-  handler: index.handler
-  timeout: 300
-  source: node_modules/@cumulus/move-granules/dist/
-  launchInVpc: true
-PostToCmr:
-  handler: index.handler
-  timeout: 300
-  memory: 256
-  logToElasticSearch: true
-  source: node_modules/@cumulus/post-to-cmr/dist/
-  useMessageAdapter: true
-  launchInVpc: true
-  envs:
-    system_bucket: '{{system_bucket}'
-```
+Once you've configured your CMR credentials, updated your workflow configuration, and updated your lambda configuration you should be able to redeploy your cumulus instance by running the following commands:
 
-**Please note**: ```FakeProcessing``` is the core provided browse/cmr generation we're using for the example.
+* ```terraform init```
```

Comment:
I would just bold this content instead of making a floating list item. Or you can combine lines 202-204 to avoid it altogether

Reply:
👍 2bf6289fe